MAUI/MauiAppWithBroker/MSALClient/PCAWrapper.cs

```diff
@@ -10,49 +11,59 @@
 namespace MauiAppWithBroker.MSALClient
 {
     /// <summary>
-    /// This is a wrapper for PCA. It is singleton and can be utilized by both application and the MAM callback
+    /// This is a wrapper for PublicClientApplication. It is singleton and can be utilized by both application and the MAM callback
     /// </summary>
     public class PCAWrapper
     {
 
         /// <summary>
-        /// This is the singleton used by consumers
+        /// This is the singleton used by consumers. Since PCAWrapper constructor does not have perf or memory issue, it is instantiated directly.
         /// </summary>
         public static PCAWrapper Instance { get; } = new PCAWrapper();
 
-        internal IPublicClientApplication PCA { get; }
-
         /// <summary>
-        /// The authority for the MSAL PublicClientApplication. Sign in will use this URL.
+        /// Instance of PublicClientApplication. It is provided, if App wants more customization.
         /// </summary>
-        private const string _authority = "https://login.microsoftonline.com/common";
+        internal IPublicClientApplication PCA { get; }
 
         // ClientID of the application in (ms sample testing)
         private const string ClientId = "858b4a09-dc31-45d3-83a7-2b5f024f99cd"; // TODO - Replace with your client Id. And also replace in the AndroidManifest.xml
 
-        //// TenantID of the organization (msidlab4.com)
-        //private const string TenantId = "f645ad92-e38d-4d1a-b510-d1b09a74a8ca"; // TODO - Replace with your TenantID. And also replace in the AndroidManifest.xml
+        // TenantID of the organization (ms sample testing)
+        private const string TenantId = "7f58f645-c190-4ce5-9de4-e2b7acd2a6ab"; // TODO - Replace with your TenantID. And also replace in the AndroidManifest.xml

```

Comment:
No TODOs please. All contsnts should be in a settings files.

Reply:
Good suggestion. Created an AppConstants class for Basic and Broker